app/webpacker/src/javascripts/commodities.js.erb

```diff
@@ -746,7 +746,7 @@ import debounce from "./debounce";
                 source: debounce(function(query, populateResults) {
                   $.ajax({
                     type: "GET",
-                    url: window.geographicalAreasPath,
+                    url: $(".path_info").data("geographicalAreasPath"),
```

Comment:
The only comment would be to use a proper api endpoint here. Will the data attrib grow here?

Reply:
Shouldn't grow beyond the extent of our current needs. I think it'll actually get removed. Not sure what you mean by a proper api endpoint? Maybe something we can discuss in person another time